src/bika/lims/browser/analyses/view.py

```diff
@@ -1088,25 +1078,21 @@ def _folder_item_instrument(self, analysis_brain, item):
             item["Instrument"] = _("Manual")
 
     def _folder_item_analyst(self, obj, item):
-        is_editable = self.is_analysis_edition_allowed(obj)
-        if not is_editable:
-            item['Analyst'] = obj.getAnalystName
-            return
-
-        # Analyst is editable
-        item['Analyst'] = obj.getAnalyst or api.get_current_user().id
-        item['choices']['Analyst'] = self.get_analysts()
```

Comment:
Can yo remember why we had this?
Maybe this view is used somewhere where we allow to set the Analyst?

Reply:
Had exactly same concern, but is not used anymore. This was introduced with https://github.com/senaite/senaite.core/commit/1138fe2c206bbd9a2433c9b6c4d2dcc2023f903e. Nowadays, "Analyst" field in analyses is never editable because it was no longer added to `allow_edit` sincef https://github.com/senaite/senaite.core/pull/1111/files#diff-0725c710c82fcbeee927a69c3605b011cd11f60283f7eca95b799f2dcddd9771L777 . I've even tested this with "field" analyses

Reply:
Cool, thanks for tracing this back to 2014:)